### ISSUE TITLE
If parameter has a forward slash, the errors are not rendered in the sidebar

### ISSUE THREAD

**Issue by jrsquared** [josdejong/jsoneditor#761]
Simple data structure for testing. I verified that ajv correctly generates the error, and if I change the key from `foo/bar` to `foobar`, it renders correctly.

```
const schema = {
  $schema: 'http://json-schema.org/draft-06/schema#',
  additionalProperties: false,
  definitions: {
    option_1: {
      additionalProperties: false,
      properties: {
        lagid: {
          maximum: 399,
          minimum: 300,
          type: 'integer',
        },
      },
      required: ['lagid'],
    },
    option_2: {
      additionalProperties: false,
      properties: {
        metric: {
          maximum: 16777214,
          minimum: 1,
          type: 'integer',
        },
      },
      required: ['metric'],
    },
  },
  description: 'test',
  patternProperties: {
    '^[a-zA-Z0-9/:-]+$': {
      oneOf: [
        {
          $ref: '#/definitions/option_1',
        },
        {
          $ref: '#/definitions/option_2',
        },
      ],
      type: 'object',
    },
  },
  title: 'a test thing',
  type: 'object',
};

const json = {
  'foo/bar': {
    lagid: 301,
  },
  foo: {
    metric: 11,
  },
};
```

**Comment by josdejong** [comment]
Thanks for reporting, I can reproduce this issue. There is indeed an error (try `editor.validateSchema.errors`) but it is not rendered in any of the modes (tree, code, ...)

**Comment by jrsquared** [comment]
It _is_ rendered in the footer, just not in the side gutter, so `jsoneditor` at least understands the errors to some extent.

**Comment by josdejong** [comment]
Yes indeed, thanks for the clarification.

**Comment by josdejong** [comment]
Fixed via 964fe3e06d9542807593e576b5b67816d2efa59d (not yet published)

**Comment by josdejong** [comment]
This is fixed now in `v6.4.0`, can you give it a try Justin?

**Comment by jrsquared** [comment]
Just tried it tonight, it works great! Thank you so much.
